tutorials/SpectroscopicDataReductionBasics/1-SpectroscopicTraceTutorial.ipynb

```diff
@@ -0,0 +1,1100 @@
+{
```

Comment:
<p>The main goal...defining the position of the signal along the detector.  (Later you call it "signal" so let's start calling it signal here.) </p><p> </p><p>This is an approximation made by inspecting the image by eye.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/508/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>revised.</p>